### ISSUE TITLE
Connectives work only when formalizing from outside to inside

### ISSUE THREAD

**Issue by terezanovot** [normativeai/frontend#23]
If formalizing from inside to outside (which is more intuitive I think), the inside connective is deleted.
It is a problem mainly if the outside formalization expects different number of inputs (operands) then there is in the whole text (sentence).

**Comment by lex-lex** [comment]
Yes, that is a technical limitation at the moment and there is no simple way around it (at least not before the tutorial). Annotation needs to be done from the outside to the inside.

**Comment by shaolintl** [comment]
This might be a nice future feature though...
